### ISSUE TITLE
Having problems downloading Naruto

### ISSUE THREAD

**Issue by doitchu** [inorichi/tachiyomi#626]
Hi!

Unfortunately, when I am trying to download Naruto in Mangafox, the last page couldn't be downloaded. I tried downloading other chapters, but the last pages really couldn't be downloaded. I tried downloading in other catalogues, but it says "error".  It happened after the latest version was released, I guess. >< I thought  the problem is my phone, but I tried using my sister's and the same thing happened. It only happens in Naruto tho, since I tried downloading Shingeki no Kyojin's latest chapter and Noragami, and both are fine.

Thank you so much!

Version: v0.4.2

**Comment by inorichi** [comment]
Most likely the manga you are trying to read is licensed in your country and mangafox blocked it. Mangafox lists all the chapters but doesn't let you read them.

**Comment by doitchu** [comment]
Oh. :c So, then it's a bit impossible for me now to read Naruto again? I am already at chapter 352 when it happened. :c Before that, I could download a lot of chapters of Naruto without any problems. ><

**Comment by wopian** [comment]
Just read Naruto on one of the other sources?

**Comment by inorichi** [comment]
Related to #478. Nothing to fix here.

**Comment by doitchu** [comment]
@wopian Hello! I also tried it, but then whenever I tried downloading, it always says "error". ;________;

**Comment by wopian** [comment]
I just searched Naruto on Mangasee, and could see page 45 of chapter 700.6 just fine.

**Comment by doitchu** [comment]
@wopian If I just open a chapter, I could also see it. But when I start downloading it, it says "error". But thanks, anyways. I appreciate you commenting here! Guess I'll just read Naruto somewhere else. Thank you!

**Comment by inorichi** [comment]
Hm... if some pages work it shouldn't be the licensed state, but something breaking the parser. I'd recommend you to read it somewhere else though.

**Comment by inorichi** [comment]
http://mangafox.me/manga/naruto/v72/c700.6/47.html

This indeed breaks the parser, *sigh*. It's a requisite that all the pages are downloaded for it to complete.

You can delete the "_tmp" in the folders of your downloads directory and they will be marked as downloaded. It's safe to do that given that the missing page is just a dummy.

**Comment by inorichi** [comment]
I introduced a workaround for this issue. It will take a while to be included in a new release though.

**Comment by doitchu** [comment]
@inorichi Hiii! I just want to say that I can read Naruto again using Tachiyomi! :D Right after I posted this issue of mine here, I don't know what happened but when I tried to download a chapter of Naruto again, it worked! I am sorry to be saying this a little too late. ;_____; I was also late to see that you have another comments in here. I am sorry! >< But I just want to thank you for the hardwork! <3 Thank you soooooo much!
